fetch.bs

```diff
@@ -7130,8 +7130,8 @@ due course.
 <!-- https://lists.w3.org/Archives/Public/public-whatwg-archive/2014Jun/thread.html#msg72 -->
 
 <p>Objects including the {{Body}} interface mixin need to define an associated
-<dfn id=concept-body-mime-type for=Body>MIME type</dfn> algorithm which takes no arguments and
-returns failure or a <a for=/>MIME type</a>.
+<dfn id=concept-body-mime-type for=Body>MIME type</dfn> algorithm which returns the result of
+<a for=Body>get the MIME type</a> with <a>this</a>.
```

Comment:
I think we should remove this, no? Callers can use "get the MIME type" directly.